extensions/wikia/Wall/WallRelatedPages.class.php

```diff
@@ -142,7 +142,14 @@ function getMessagesRelatedArticleTitles( $messageIds, $orderBy = 'order_index'
 	}
 
 
-	public function getArticlesRelatedMessgesSnippet( $pageId, $messageCount, $replyCount ) {
+	/**
+	 * TODO: Used by RelatedForumDiscussionController only, move there?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)